hadoop-ozone/recon/src/test/java/org/apache/hadoop/ozone/recon/api/TestNSSummaryEndpointWithLegacy.java

```diff
@@ -145,6 +146,12 @@ public class TestNSSummaryEndpointWithLegacy {
   private static final String KEY_NINE = "dir5/file9";
   private static final String KEY_TEN = "dir5/file10";
   private static final String KEY_ELEVEN = "file11";
+  private static final String KEY_TWELVE = "file12";
```

Comment:
Could you also update the tree diagram for `TestNSSummaryEndpointWithLegacy` with the new changes. I think we could implement a similar diagram style like present in `TestNSSummaryEndpointWithOBSAndLegacy` to show case the file hierarchy.